.vscode/launch.json

```diff
@@ -1,76 +0,0 @@
-{
-	// Use IntelliSense to learn about possible Node.js debug attributes.
-	// Hover to view descriptions of existing attributes.
-	// For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
-	"version": "0.2.0",
-	"configurations": [
-		{
-			"type": "node",
-			"request": "launch",
-			"cwd": "${workspaceRoot}",
-			"sourceMaps": true,
-			"name": "Launch CLI (Node 6+)",
-			"program": "${workspaceRoot}/lib/nativescript-cli.js",
-
-			// example commands
-			"args": [ "create", "cliapp"]
-			// "args": [ "platform", "add", "android@1.3.0", "--path", "cliapp"]
-			// "args": [ "platform", "remove", "android", "--path", "cliapp"]
-			// "args": [ "plugin", "add", "nativescript-barcodescanner", "--path", "cliapp"]
-			// "args": [ "plugin", "remove", "nativescript-barcodescanner", "--path", "cliapp"]
-			// "args": [ "build", "android", "--path", "cliapp"]
-			// "args": [ "run", "android", "--path", "cliapp"]
-			// "args": [ "debug", "android", "--path", "cliapp"]
-			// "args": [ "livesync", "android", "--path", "cliapp"]
-			// "args": [ "livesync", "android", "--watch", "--path", "cliapp"]
-		},
-		{
-			// in case you want to debug a single test, modify it's code to be `it.only(...` instead of `it(...`
```

Comment:
it won't be that's the idea, every dev can have his own configuration. 
You'll still have your configuration, but it won't be in github.
Maybe I should have mentioned that you should, save your `launch.json` configuration somewhere, then pull master, then put bach the `launch.json` configuration.